### DISCUSSION TITLE
Welcome to OpenScienceGame Discussions!

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [GeorgiaHCA/OpenScienceGame#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GeorgiaHCA** [comment]
Hiya everyone! I've been asked about a possible application for this game to adapt it as a therapeutic tool to encourage pro-social behaviours. It seems like a positive use! I'll add more info when I have it. Obviously it's been a while since we made the game, but I do think it's really cool and it would be lovely to see it being used. I was also thinking about running it again with my lab at The Turing and see what they all think, as I believe it has so much potential. How would you all like to be credited/would you like to be involved more? Hope you are all doing great!

**Comment by vvaneli** [comment]
Hey @GeorgiaHCA! Agree that our game definitely has lots of underused and unexplored potential, and it’d be great to have it used more. I might be able to help with some aspects depending on what it is and time demands.
